### PR TITLE
Using flatted instead of circular-json

### PR DESCRIPTION
**Pull request by sainianubhav**
Flatted is successor of circular-json module. 

https://github.com/WebReflection/circular-json#the-future-of-this-module-is-called-flatted

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
       "resolved": "https://registry.npmjs.org/accepts/-/accepts-1.3.5.tgz",
       "integrity": "sha1-63d99gEXI6OxTopywIBcjoZ0a9I=",
       "requires": {
-        "mime-types": "2.1.18",
+        "mime-types": "~2.1.18",
```
Comment (sainianubhav) [inline package-lock.json]: I have no clue why this happens on my machine. `npm install` always updates packaeg-lock and moves dependency versions from exact to minor or patch level updatable ones.

```

$ node -v
v8.11.3

$ npm -v
6.4.1

```

Comment (sainianubhav) [reply package-lock.json]: Search for `es6-symbol`. This goes from `3.1.1` to `~3.1.1` and `^3.1.1`

Comment (kumarrishav) [reply package-lock.json]: For ref: https://github.com/npm/npm/issues/20891
https://github.com/npm/npm/issues/20434#issuecomment-392421024

**Comment by sainianubhav** [comment]
@grawk all done.

**Comment by grawk** [comment]
@sainianubhav next time you submit a PR feel free to add yourself to the contributors block in `package.json`

**Comment by sainianubhav** [comment]
@grawk Thanks. Let me pull my weight first.

**Pull request merged**